docs/lib/datastore/fragments/android/sync/50-selectiveSync.md

```diff
@@ -0,0 +1,284 @@
+## Selectively syncing a subset of your data
+
+By default, DataStore downloads the entire contents of your cloud data source to your local device.
+The max number of records that will be stored is configurable [here](~/lib/datastore/conflict.md).
+
+You can utilize selective sync to only persist a subset of your data instead.
+
+Selective sync works by applying predicates to the base and delta sync queries, as well as to incoming subscriptions.
+
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
```

Comment:
Is it worth throwing the `RxAmplify` variant here? I know it isn't much different, but for symmetry with the other examples.

Reply:
added in the latest commit!